torch/csrc/jit/python/init.cpp

```diff
@@ -885,13 +886,125 @@ void initJITBindings(PyObject* module) {
             docstring << "  " << op->schema() << "\n";
           }
 
-          return py::cpp_function(
-              [operations](py::args args, py::kwargs kwargs) {
```

Comment:
Just a 10 mile view comment here: this is a lot more code than I would have expected here!

Reply:
Sure, refactored a bit, and should be easy to understand now.

Reply:
OK, this is much better, thank you!